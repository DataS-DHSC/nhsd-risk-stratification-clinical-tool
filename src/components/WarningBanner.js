import React from 'react';

const WarningBanner = () => (
  <div
    className="nhsuk-global-alert"
    id="nhsuk-global-alert"
    role="complementary"
  >
    <div className="nhsuk-width-container">
      <div className="nhsuk-grid-row">
        <div className="nhsuk-grid-column-full">
          <div className="nhsuk-global-alert__content">
            <div className="nhsuk-global-alert__message">
              <p>
                <strong>Important information:</strong> This is a prototype for
                demonstration purposes, not a functioning clinical tool.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default WarningBanner;
