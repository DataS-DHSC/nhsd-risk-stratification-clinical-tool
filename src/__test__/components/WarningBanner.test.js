import React from 'react';
import { render } from '@testing-library/react';
import WarningBanner from '../../components/WarningBanner';

const setup = () => render(<WarningBanner />);

describe('WarningBanner', () => {
  it('should match snapshot ', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
