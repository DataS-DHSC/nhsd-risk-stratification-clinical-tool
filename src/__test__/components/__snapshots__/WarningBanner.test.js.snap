// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WarningBanner should match snapshot  1`] = `
<DocumentFragment>
  <div
    class="nhsuk-global-alert"
    id="nhsuk-global-alert"
    role="complementary"
  >
    <div
      class="nhsuk-width-container"
    >
      <div
        class="nhsuk-grid-row"
      >
        <div
          class="nhsuk-grid-column-full"
        >
          <div
            class="nhsuk-global-alert__content"
          >
            <div
              class="nhsuk-global-alert__message"
            >
              <p>
                <strong>
                  Important information:
                </strong>
                 This is a prototype for demonstration purposes, not a functioning clinical tool.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
